File: Area and circumference of Circle.c
// Write a program to find Area and circumference of Circle


#include<stdio.h>
float getNumberFromUser(){
    float number;
    scanf("%f", & number );
    return number;
}

float findTheArea(float r){
return 3.14*(r*r);
}

float findTheCircumference(float r){
    return 2*(3.14*r);
}

void main(){
    printf("enter the radius of circle ");
    float r = getNumberFromUser();
    float area = findTheArea(r);
    printf("\n area is %f ", area);
    float circumference = findTheCircumference(r);
    printf("\n cricumference is %f ", circumference);
}

File: JavaScript/Area and circumference of Circle.js
// Write a program to find Area and circumference of Circle

const prompt = require("prompt-sync")();

const x = parseInt(prompt("Enter the Radius of circle = "));

const Area = 3.12 * x * x
const Circumference = 2 * 3.14 * x

console.log("Enter the area of circle = ",Area , "\n Enter the circumfernce of circle = ",Circumference );
